src/shell/views/Shell/AIDrawer.tsx

```diff
@@ -0,0 +1,493 @@
+
+[
+  {
+    "type": "NO_MATCH",
+    "payload": {
+      "value": "No field to modify was found. Closest might be '<closestMatch>'"
+    }
+  }
+]
+
+---
+
+### Guidelines for 'value':
+- If the prompt asks for **titles, content, or descriptions**, generate high-quality content—not just restating the prompt.
+- Adapt tone and language properly.
+- If multiple fields seem applicable, use the prompt's cues to prioritize.
+- Never include the field context, internal notes, or any explanation in the output.
+
+---
+
+### Available refKeys:  
+[${registryKeys}]
+
+Context for refKeys (for your matching logic only – **never include this in output**):  
+${JSON.stringify(
+  registryKeys.map((x) => `"${x}": "${JSON.stringify(refRegistry[x].context)}"`)
+)}
+
+---
+
+⚠️ **Important Output Rules:**
+- Return only valid JSON.
+- Output must be a single top-level array of valid action objects.
+- No comments, markdown, or extra text.
+`;
+
+  useEffect(() => {
+    if (!aiResponse) return;
+
+    try {
+      const cleaned = aiResponse.data.replace(/```json|```/g, "").trim();
+      const parsed = JSON.parse(cleaned);
+      const responsesArray = Array.isArray(parsed) ? parsed : [parsed];
+
+      const animateTyping = (response: any, indexInArray: number) => {
+        const fullText = response.payload.value;
+        let currentText = "";
+        let currentIndex = 0;
+
+        const interval = setInterval(() => {
+          currentText += fullText[currentIndex];
+
+          setResponses((prev) => {
+            const updated = [...prev];
+            if (updated[indexInArray]) {
+              updated[indexInArray] = {
+                ...response,
+                payload: { ...response.payload, value: currentText },
+              };
+            }
+            return updated;
+          });
+
+          currentIndex++;
+          if (currentIndex >= fullText.length) clearInterval(interval);
+        }, 20);
+      };
+
+      responsesArray.forEach((response) => {
+        if (response.type === "USER_INPUT") {
+          setResponses((prev) => [...prev, response]);
+          return;
+        }
+
+        // Push placeholder and animate after it's in state
+        setResponses((prev) => {
+          const newIndex = prev.length;
+          const newResponses = [
+            ...prev,
+            {
+              ...response,
+              payload: { value: "", refKey: response.payload.refKey },
+            },
+          ];
+          setTimeout(() => animateTyping(response, newIndex), 0);
+          return newResponses;
+        });
+        if (autoApply && response.type === "SET_VALUE") {
+          enqueueAction({
+            type: response.type,
+            payload: {
+              refKey: response.payload.refKey,
+              value: response.payload.value,
+            },
+          });
+        }
+      });
+    } catch (error) {
+      console.error("Error parsing AI response", error);
```

Comment:
Remove comment if unused

Reply:
Oh I meant the console.error, unless that's actually needed and not just for debugging purposes